docs/getting-started.md

```diff
@@ -244,6 +244,15 @@ available hardware.
 
 {% include /style/begin-section.html %}
 
+## Step 5.1: Update `apt`
```

Comment:
I think this needs to be after step 6 since an SSH connection is required to update the `sources.list` file.